### DISCUSSION TITLE
vue 项目中 postcss-px-to-viewport 导致 tailwindcss 失效

### DISCUSSION THREAD

**Discussion by FinnIsMe** [tailwindlabs/tailwindcss#12418] [Help]
postcss: {
        plugins: [
          require('postcss-px-to-viewport')({
            unitToConvert: 'px', // 要转化的单位
            viewportWidth: 1920, // UI设计稿的宽度
            unitPrecision: 3, // 转换后的精度，即小数点位数
            propList: [
              '*'
            ], // 指定转换的css属性的单位，*代表全部css属性的单位都进行转换
            viewportUnit: 'vw', // 指定需要转换成的视窗单位，默认vw
            fontViewportUnit: 'vw', // 指定字体需要转换成的视窗单位，默认vw
            landscapeUnit: 'vh', // 横屏时使用的单位
            landscapeWidth: 1920, // 横屏时使用的视口宽度
            selectorBlackList: [], // 指定不转换为视窗单位的类名
            minPixelValue: 1, // 默认值1，小于或等于1px则不进行转换
            mediaQuery: false, // 是否在媒体查询的css代码中也进行转换，默认false
            replace: true // 是否转换后直接更换属性值
            // landscape: false, // 是否处理横屏情况
            // exclude: /(\/|\\)(node_modules)(\/|\\)/,
          })
        ]
      }

**Comment by xie392** [comment]
![image](https://github.com/tailwindlabs/tailwindcss/assets/61953437/72633a86-29d6-4100-a991-9626d06ef0da)

**Reply by xie392** [reply]
可以这样写，配置你的vite.config.js

**Comment by zuowendong** [comment]
这个插件不支持，可以试试 [postcss-px-to-viewport-tailwind](https://www.npmjs.com/package/postcss-px-to-viewport-tailwind)
